src/test/java/org/llorllale/cactoos/matchers/InputHasContentTest.java

```diff
@@ -46,7 +46,7 @@
     @Test
```

Comment:
`StringLiteralsConcatenationCheck` can be removed here

Reply:
@ilyakharlamov fixed